### PR TITLE
fix build of pivottable using gxquery

### PR DESCRIPTION
**Pull request by c-milocanovich**
* Include queryViewerId in the service Context
* Update version reporting-api
* Minor improvements to storybook documents

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -57,6 +57,7 @@
     "eslint-plugin-storybook": "^0.6.14",
     "jest": "^27.4.5",
     "jest-cli": "^27.4.5",
+    "lit": "^3.1.2",
```
Comment (ncamera) [inline package.json:60]:
```suggestion
```

**Review by ncamera — CHANGES_REQUESTED** [review]
LGTM! Just a small nit.

File: package.json (hunk 1 above)
Comment (c-milocanovich) [reply package.json:60]: "Iit" is used for storybook to render the components.
I added it as dev dependency

```
import { html } from 'lit';
...
const meta: Meta<HTMLGxQueryViewerControllerElement> = {
  component: "gx-query-viewer-controller",
  decorators: [(story, context) => html`<gx-query-viewer type="${context.args.type}">${story()}</gx-query-viewer>`],

```

**Review by ncamera — APPROVED** [review]
👍

**Pull request merged**